app/recipe/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+
+class Ingredient(models.Model):
+    """
+    Ingredient used in a recipe
+    """
+    name = models.CharField(max_length=255)
+    recipe = models.ForeignKey(
+        'Recipe',
+        on_delete=models.CASCADE,
+        related_name='ingredients'
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class RecipeManager(models.Manager):
+    def create_recipe(self, **kwargs):
```

Comment:
I would be more explicit with the parameters, I would separate the ingredients in a different field something like 
```suggestion
    def create_recipe(self, recipe_data: dict, ingredients: List[dict]):
```

Reply:
Changed, you're right, that way readability is improved.